prisma/schema.prisma

```diff
@@ -88,7 +88,7 @@ model User {
   id                  Int                  @id @default(autoincrement())
   username            String?              @unique
   name                String?
-  email               String               @unique
+  email               String               @unique @db.Citext
```

Comment:
This makes the column case insensitive, which will prevent mixed case at DB level.